docs/book/src/cronjob-tutorial/testdata/project/internal/controller/suite_test.go

```diff
@@ -74,11 +74,26 @@ var _ = BeforeSuite(func() {
 
 	ctx, cancel = context.WithCancel(context.TODO())
 
+	var err error
+	/*
+		First, the autogenerated test code will add the CronJob Kind schema to the default client-go k8s scheme.
+		This ensures that the CronJob API/Kind will be used in our test controller.
```

Comment:
```suggestion
    The CronJob Kind is added to the client-go Kubernetes scheme.
    This ensures that the CronJob API is registered with the scheme, allowing the test controller to recognize and interact with CronJob resources.
```

Add the schema does not ensure that the Kind will be used
That is required in order to allow we use it in the test.
Only the Core Types are added to the scheme by default.

Reply:
will add to docs PR